request-manager/api/api.go

```diff
@@ -204,7 +205,9 @@ func (api *API) createRequestHandler(c echo.Context) error {
 	// Run (non-blocking)
 
 	if err := api.rm.Start(req.Id); err != nil {
-		api.rm.Fail(req.Id)
+		if err := api.rm.FailPending(req.Id); err != nil {
+			log.Errorf("error starting request in RM: %s", err)
```

Comment:
include the request id in the error message too please — will make it easier to see what exactly happened.